extensions/amp-consent/amp-consent.md

```diff
@@ -388,77 +401,77 @@ Example:
 
 **Note**: For some vendors, the `data-block-on-consent` attribute only works in conjunction with `<amp-consent>`. Consult with your ad or analytics vendor for implementation details.
 
-##### How can I gather consent from all users on AMP pages? 
+##### How can I gather consent from all users on AMP pages?
 
-Use the `<amp-consent>` component which allows configuring a custom UI with `Accept`, `Reject` and `Dismiss` states. It is up to each amp-component vendor how they interpret these states, so please read your vendor's documentation carefully. 
+Use the `<amp-consent>` component which allows configuring a custom UI with `Accept`, `Reject` and `Dismiss` states. It is up to each amp-component vendor how they interpret these states, so please read your vendor's documentation carefully.
 
 Note that you must configure either the [`promptIfUnknownForGeoGroup`](#promptifunknownforgeogroup) attribute or the checkConsentHref request.
 
-##### How can I stop showing all of my content to users from the EEA?  
+##### How can I stop showing all of my content to users from the EEA?
 
 You could consider hiding all your content for EEA users by implementing [`<amp-geo>`](https://www.ampproject.org/docs/reference/components/amp-geo). See [this blog post](https://www.ampproject.org/latest/blog/dynamic-geo-personalization/) for more details.
 
-##### How can I target consent for only EEA users?  
+##### How can I target consent for only EEA users?
 
 Use [`<amp-geo>`](https://www.ampproject.org/docs/reference/components/amp-geo) to configure a country group and hook up the country group to `promptIfUnknownForGeoGroup` attribute in `<amp-consent>`. If the user accesses the AMP page from a country that's configured in the list, the appropriate consent UI will be invoked.
 
 ##### Can I make the consent UI blocking?
 
-Yes. The UI is not prescriptive. If you do end up providing a non-blocking version of the UI, dismissing the UI will lead to a `dismiss` state. It is up to each vendor (ads & analytics) how they would process `accept`, `reject` and `dismiss`. 
+Yes. The UI is not prescriptive. If you do end up providing a non-blocking version of the UI, dismissing the UI will lead to a `dismiss` state. It is up to each vendor (ads & analytics) how they would process `accept`, `reject` and `dismiss`.
 
-##### What is `checkConsentHref`? And why is it mandatory? 
+##### What is `checkConsentHref`? And why is it mandatory?
```

Comment:
update this

Reply:
Ha found we explained that it's not mandatory with `promptIfUnknownForGeoGroup` right below. https://github.com/ampproject/amphtml/blob/master/extensions/amp-consent/amp-consent.md#what-is-checkconsenthref-and-why-is-it-mandatory